### PR TITLE
User orders

### PR DESCRIPTION
**Pull request by hopesgit**
Lordy this one was big. 

Add functionality for :
- User Profiles Displaying orders
- Merchant User profiles displaying orders with the ability to fulfill items
- Admin dashboard allows an Admin to view all orders, sorted by status, and ship the packaged ones
- Users can't cancel shipped orders
- Checking out removes items and cancelling orders restores them to inventory

All tests passing

This covers stories 27-33

### CODE REVIEW

**Review by Todd-Estes — APPROVED** [review]

**Comment by Todd-Estes** [comment]
Great work, Hope!!!

File: app/views/admin/dashboard/show.html.erb [hunk 1]
```diff
@@ -1 +1,70 @@
 <h1>This is a space for the All-Powerful to rest!</h1>
+<br>
+
+<section id="packaged-orders">
+  <h2>Packaged Orders:</h2>
+  <% if @orders.packaged_orders.empty? %>
```
Comment (GeorgieGirl24) [inline app/views/admin/dashboard/show.html.erb:6]: Feels like there is a more efficient way to do this....

File: db/migrate/20201101210533_add_status_to_item_orders.rb [hunk 2]
```diff
@@ -0,0 +1,5 @@
+class AddStatusToItemOrders < ActiveRecord::Migration[5.2]
+  def change
```
Comment (GeorgieGirl24) [inline db/migrate/20201101210533_add_status_to_item_orders.rb:2]: great! We needed this attribute

File: spec/features/sessions/logout_spec.rb [hunk 3]
```diff
@@ -17,7 +17,6 @@
       fill_in("Email Address", with: "#{@user_3.email_address}")
       fill_in("Password", with: @user_3.password)
       click_button("Submit")
-      # save_and_open_page
```
Comment (GeorgieGirl24) [inline spec/features/sessions/logout_spec.rb:20]: we need to remove this

**Review by GeorgieGirl24 — APPROVED** [review]

**Pull request merged**